templates/helm.yml

```diff
@@ -96,11 +96,17 @@
       helm search repo current-repo/"${CHART_NAME}"
       echo
 
+      LATEST=$(helm search repo current-repo/"${CHART_NAME}" | tail -n 1 | awk '{print $2}')
+
+      if [ -z "${VERSION}" ]; then
```

Comment:
che ne pensi se la rinominassimo in CHART_VERSION per renderla meno generica?

Reply:
magari aggiungiamo anche un messaggio dopo l'if mostrando la versione usata del chart. Es. "Deploying ${APP_NAME} with chart version ${CHART_VERSION}"

Reply:
mi piace